### DISCUSSION TITLE
CRUD - Authentication Demo

### DISCUSSION THREAD

**Discussion by pchemguy** [sqlpage/SQLPage#466] [Show and tell]
The demo is presently hosted in a separate [repo](https://github.com/pchemguy/SQLPage-Demo/tree/main/CRUD%20-%20Authentication)

## Features

- Basic authentication with cookie-based session management
- View/Edit forms for a database table with CRUD operations implemented
- Current page tracking for appropriate returns after Login/Logout and database modification operations
- Selective execution of sections in an external SQL module via SET-based flags set before run_sql()
- Custom shell component and buttons in the top menu

![](https://raw.github.com/pchemguy/SQLpage/crud_auth/examples/crud-authentication/www/img/table_view.png)
![](https://raw.github.com/pchemguy/SQLpage/crud_auth/examples/crud-authentication/www/img/detail_view.png)
![](https://raw.github.com/pchemguy/SQLpage/crud_auth/examples/crud-authentication/www/img/delete_confirmation.png)

**Comment by lovasoa** [comment]
Thank you very much, @pchemguy ! Just read the README on your repo, it's a great technical introduction to SQLPage. And the CRUD example is very detailed and useful. Would you be interested in moving it to https://github.com/lovasoa/SQLpage/tree/main/examples ?

**Reply by pchemguy** [reply]
Sure, I will make a pull request
